lib/util/UserInteractor.js

```diff
@@ -678,6 +678,33 @@ class UserInteractor {
             1);
     }
 
+    static processYesNoDefaultAccountChoice(message, continueChoiceHandler, continueChoiceHandlerParams, confirmed, printMsg = true) {
+        if(confirmed) {
+            continueChoiceHandlerParams['def'] = true;
```

Comment:
Should we not be using the constant `LOGIN_DEFAULT` defined above?

Reply:
Which one?

Reply:
`continueChoiceHandlerParams[LOGIN_DEFAULT] = true`

Reply:
Sorry, I still can't find that constant!